app/javascript/components/student/iterations-list/RepresenterFeedback.tsx

```diff
@@ -21,3 +26,16 @@ export const RepresenterFeedback = ({ html, author }: Props): JSX.Element => {
     </div>
   )
 }
+
+function EditedBy({
+  author,
+  editor,
+}: Pick<Props, 'author' | 'editor'>): JSX.Element | null {
+  if (editor && editor.name !== author.name) {
```

Comment:
Maybe early return is nicer here?
`if (!editor || editor.name == author.name) { return null }`